OpenApiSpecs/exchangeSetService_OpenApi_definition_internal.yaml

```
@@ -0,0 +1,1291 @@
+                  The event payload. This specification does not place any restriction on the type of this information. It is encoded into a media format which is specified by the datacontenttype attribute (e.g. application/json), and adheres to the dataschema format when those respective attributes are present.
+      
+                  The `uk.co.admiralty.avcsData.exchangeSetCreated.v1` event payload is described by "#/components/schemas/exchangeSetResponse".
+                allOf:
+                  - $ref: "#/components/schemas/exchangeSetResponse"
+            example:
+              specversion: "1.0"
+              type: "uk.co.admiralty.avcsData.exchangeSetCreated.v1"
+              source: " https://exchangeset.admiralty.co.uk/avcsData"
+              id: "2f03a25f-28b3-46ea-b009-5943250a9a41"
+              time: "2021-02-17T14:04:04.4880776Z"
+              subject: "Requested S-100 Exchange Set Created"
+              datacontenttype: "application/json"
+              data:
+                _links:
+                  exchangeSetBatchStatusUri:
+                    href: "http://fss.ukho.gov.uk/batch/7b4cdf10-adfa-4ed6-b2fe-d1543d8b7272/status"
+                  exchangeSetBatchDetailsUri:
+                    href: "http://fss.ukho.gov.uk/batch/7b4cdf10-adfa-4ed6-b2fe-d1543d8b7272"
+                  exchangeSetFileUri:
+                    href: "http://fss.ukho.gov.uk/batch/7b4cdf10-adfa-4ed6-b2fe-d1543d8b7272/files/exchangeset123.zip"
+                  errorFileUri:
+                    href: "http://fss.ukho.gov.uk/batch/7b4cdf10-adfa-4ed6-b2fe-d1543d8b7272/files/error.txt"
+                exchangeSetUrlExpiryDateTime: "2021-02-17T16:19:32.269Z"
+                requestedProductCount: 4
+                exchangeSetProductCount: 1
+                requestedProductsAlreadyUpToDateCount: 0
+                requestedProductsNotInExchangeSet:
+                  [
+                    { productName: "101GB40079ABCDEFG", reason: "invalidProduct" },
+                  ]
+                fssBatchId: "7b4cdf10-adfa-4ed6-b2fe-d1543d8b7272" 
+    ####################################
+    #      Security Objects      #
+    ####################################
+
+  securitySchemes:
+    jwtBearerAuth:
+      type: http
+      scheme: bearer
+      bearerFormat: JWT
+
+    ######################
+    #      callbacks     #
+    ######################
+
+  callbacks:
+    fulfilmentResponse:
+      '{$request.query.callbackUri}':
+        post:
+          summary: Notify the Exchange Set requestor that this is now ready to download on the File Share Service.
+          description: |
+            Once the Exchange Set has been committed on File Share Service, a notification will be sent to the callbackURI (if specified).  
+
+            ## Data:
+
+            The data for the notification will follow the CloudEvents 1.0 standard, with the data portion containing the same Exchange Set data as the response to the original API request.
```

To disambiguate from the S-100 Exchange Set response, suggest adding a link to the payload schema

```suggestion
            The data for the notification will follow the CloudEvents 1.0 standard, with the data portion containing the same Exchange Set data as the response to the original API request ( $ref: "#/components/schemas/exchangeSetResponse" ).
```

---

Updated.